Ambit/Source/Ambit/Actors/Spawners/SpawnerBase.cpp

```diff
@@ -353,20 +350,20 @@ void ASpawnerBase::SpawnActorsAtTransforms(
                 // If actor could not be spawned at surface level,
                 // we want to sweep it to the surface and check for collision along the way
                 // before enabling SimulatePhysics
-                if (SpawnedActor->GetActorLocation() 
-                    != Transform.GetLocation()) 
+                if (SpawnedActor->GetActorLocation()
+                    != Transform.GetLocation())
                 {
                     PhysicsComponent->SetWorldLocation(
                         Transform.GetLocation(),
-                        true, nullptr, 
+                        true, nullptr,
                         ETeleportType::ResetPhysics);
                 }
             }
             // Check for any overlaps and verify that overlaps are not beyond the surface
             // of the overlapping actor
             TArray<UPrimitiveComponent*> OverlappingComponents;
             SpawnedActor
```

Comment:
Strange wrapping here. Lines 365 and 366 are only 74 chars total. Why is it getting wrapped?